### ISSUE TITLE
FYI: Retiring formula in Homebrew

### ISSUE THREAD

**Issue by javian** [w00fz/xdebug-osx#15]
Since the php tap is being merged into homebrew-core we are reviewing the most popular formulas. We are not able to migrate the formula for this repo over since it needs to be adapted to how php is set up and that there no longer is an xdebug formula once the old tap is retired.

**Comment by w00fz** [comment]
Very sad but thank you for letting me know!

**Comment by gpressutto5** [comment]
Any news?

**Comment by javian** [comment]
@w00fz will probably accept PRs if anyone is willing to get their hands dirty =)

**Comment by endelwar** [comment]
@javian I've just pushed a PR for this, do you mind testing it? https://github.com/w00fz/xdebug-osx/pull/16

**Comment by w00fz** [comment]
Is there anyway we can not hardcode the 7.2 version and pull that information from brew?

**Comment by endelwar** [comment]
I think that the hardcoded php version is resolved with last push.
@javian any change to reconsider xdebug-osx inclusion in homebrew-core once #16 get merged and a new release is done?

**Comment by w00fz** [comment]
Thank you @endelwar !

**Comment by javian** [comment]
@endelwar its a matter of someone writing a formula for it which should be pretty simple and then creating test block for it which usually is the trickier part depending on the software.

**Comment by webflo** [comment]
Related PR: Use php@version as service name

**Comment by webflo** [comment]
Fixed in https://github.com/w00fz/xdebug-osx/pull/18
